portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Calls functions to fetch the state of the page
 function runServlets() {
```

Comment:
nit: I would advise against naming this as "runServlets" since technically the backend server could be built on something else other than servlets.  Maybe something like "init()" would work well here since these are methods you'd run to build the initial page, or "runFetchers" since you are fetching multiple items from the server